iso15118/secc/controller/interface.py

```diff
@@ -699,3 +700,14 @@ async def get_15118_ev_certificate(
         - ISO 15118-20 and ISO 15118-2
         """
         raise NotImplementedError
+
+    @abstractmethod
+    async def session_stop(self, action: SessionStopAction) -> None:
```

Comment:
I think it might be better to be more explicit about the intention of this method - say like update_data_link() - as session_stop() is likely to be misunderstood to be a method for stopping a 15118 session.

Reply:
Done